pkgs/development/libraries/catboost/default.nix

```diff
@@ -0,0 +1,114 @@
+{ lib
+, config
+, stdenv
+, fetchFromGitHub
+, cmake
+, libiconv
+, llvmPackages
+, ninja
+, openssl
+, python3
+, ragel
+, yasm
+, zlib
+, cudaSupport ? config.cudaSupport or false
+, cudaPackages ? {}
```

Comment:
Isn't that always defined?

Reply:
I'm not sure if this is necessary.
I followed the [CUDA documentation](https://nixos.org/manual/nixpkgs/stable/#cuda).

Reply:
Not necessary since https://github.com/NixOS/nixpkgs/pull/224068/files#diff-1977c7748af8b43b92093d2383e23e88c4df10a702578fbe885675a0956a2f1f